### ISSUE TITLE
评测代码应该是有问题的

### ISSUE THREAD

**Issue by nwf5d** [xmxoxo/BERT-train2deploy#16]
使用提供的评测数据，得到的评测结果有问题。

> eval_accuracy = 0.86040765
eval_f1 = 0.9527646
eval_loss = 0.5360181
eval_precision = 0.9510234
eval_recall = 0.95451

在precision和recall均在0.95时，accuracy理论上也在0.95左右
作者给出的评测代码对于多分类情况同样也是有问题的。
另外，因为tensorflow的tf.metrics实现的问题，在评测数据量较大时计算也会有问题。

**Comment by nwf5d** [comment]
如果能有评测结果的文件(格式可以是：正确标签\t预测标签\t正文)，使用sklearn的metrics包也可以得到正确结果。
发现output目录下有`eval.tf_record`文件，估计只是方便后续处理生成的中间结果文件
有没有方法可以输出最终的评测结果文件呢？

**Comment by tengben0905** [comment]
precision = TP / (TP + FP)
recall = TP / (TP + FN)
accuracy = (TP + TN) / (TP + FP + TN + FN)

> 在precision和recall均在0.95时，accuracy理论上也在0.95左右
  
这个是怎么算出来的？
